src/main/java/com/altenheim/kalender/controller/logicController/GoogleAPIController.java

```diff
@@ -14,9 +14,9 @@
 
 public class GoogleAPIController implements IGoogleAPIController
 {
-    private static String FINDPLACEQUERY= "https://maps.googleapis.com/maps/api/place/findplacefromtext/json?input=%s&inputtype=textquery&fields=place_id&key=%s";
-    private static String OPENINGHOURSQUERY = "https://maps.googleapis.com/maps/api/place/details/json?place_id=%s&fields=opening_hours&key=%s";
-    private static String FINDDESTINATIONSQUERY= "https://maps.googleapis.com/maps/api/distancematrix/json?origins=%s&destinations=%s&key=%s";
+    private static final String FINDPLACEQUERY= "https://maps.googleapis.com/maps/api/place/findplacefromtext/json?input=%s&inputtype=textquery&fields=place_id&key=%s";
+    private static final String OPENINGHOURSQUERY = "https://maps.googleapis.com/maps/api/place/details/json?place_id=%s&fields=opening_hours&key=%s";
+    private static final String FINDDESTINATIONSQUERY= "https://maps.googleapis.com/maps/api/distancematrix/json?origins=%s&destinations=%s&key=%s";
```

Comment:
Die Wildcard %s muss hier mit dem API Key ersetzt werden, der aber nicht öffentlich sein darf. Im Prototyp hatte ich das mit drin weil das repo auf private war, jetzt ist es public und ich hab den Key hier (und auch bei Google natürlich gelöscht). Im Produktivcode muss der Key in den Settings liegen in einer File die von Git nicht getrackt wird und natürlich auch nicht reverse engineered werden kann, also als Hashvalue (Java hat da eigene Klassen für sowas).